src/main.cc

```diff
@@ -1164,24 +1162,32 @@ ParseRateLimiterResourceOption(const std::string arg)
 std::tuple<std::string, std::string, std::string>
 ParseBackendConfigOption(const std::string arg)
 {
-  // Format is "<backend_name>,<setting>=<value>"
+  // Format is "<backend_name>,<setting>=<value>" for single
```

Comment:
for backend specific configs/settings and .. for backend agnostic configs/settings

Reply:
Updated comment to use 'specific' and 'agnostic' respectively.